### PR TITLE
[onert/cpu] Append doxygen for ExternalTensor class

### PR DESCRIPTION
**Pull request by YongseopKim**
Append doxygen for ExternalTensor class

ONE-DCO-1.0-Signed-off-by: Yongseop Kim <yons.kim@samsung.com>

### CODE REVIEW

**Comment by YongseopKim** [comment]
PTAL @wateret

File: runtime/onert/backend/cpu/Tensor.h [hunk 1]
```diff
@@ -29,8 +29,11 @@ namespace cpu
 
 using Tensor = cpu_common::Tensor;
 
-// Tensor which has data from external. To support this, assume below things
-// no padding, always NHWC layout, constant tensor and not dynamic
+/**
+ * @brief Class that uses data from external. This class uses the data from external
```
Comment (hyunsik-yoon) [inline runtime/onert/backend/cpu/Tensor.h:33]: how about writing `external` to the following ? To me, `external` is difficult to understand.
```
 * @brief Class that uses data from external memory that is not managed by a backend. For example, ExternalTensor's data pointer can point to mmapped area of tflite or circle file.
```

Comment (YongseopKim) [reply runtime/onert/backend/cpu/Tensor.h:33]: That's very good description. I'll fix like it. Thanks.

**Comment by YongseopKim** [comment]
PTAL @hyunsik-yoon and @wateret

**Comment by hyunsik-yoon** [comment]
@nnfw-bot test cross-runtime-test-debug

File: runtime/onert/backend/cpu/Tensor.h (hunk 1 above)
Comment (wateret) [reply runtime/onert/backend/cpu/Tensor.h]: I think "mmaped area" is too narrow. As this class is a wrapper of `ir::Data`, not just `ir::MMapedData`. The description should be more general.

Comment (YongseopKim) [reply runtime/onert/backend/cpu/Tensor.h]: @wateret 
> I think "mmaped area" is too narrow. As this class is a wrapper of ir::Data, not just ir::MMapedData. The description should be more general.
Oh yes, you're right. The tensor could be all of types' ir::Data. I'll fix it soon.

**Comment by YongseopKim** [comment]
PTAL @hyunsik-yoon and @wateret

**Comment by YongseopKim** [comment]
@nnfw-bot test cross-runtime-test-release

**Comment by YongseopKim** [comment]
@nnfw-bot test cross-runtime-test-debug

**Comment by YongseopKim** [comment]
ping @Samsung/nnfw_committers

**Pull request merged**